### DISCUSSION TITLE
Filament jam

### DISCUSSION THREAD

**Discussion by szimszon** [Guilouz/Creality-Helper-Script-Wiki#637] [General]
I wonder if somebody could help me.

Do anybody else experience weird jams on prints?

With the very same gcode almost all second prints jams at layer 1 or layer 2. After I unload and load a filament everything works again. Really frustrating. Latest firmware and extruder.

There is no clog in hotend I can just unload the filament from screen...

**Comment by pauloeduardogodoy** [comment]
Are you removing the top cover during PLA or PETG long prints?

**Reply by szimszon** [reply]
yes.

Everything is the same and the second print jams the filament in extruder.

**Reply by szimszon** [reply]
[gcode.zip](https://github.com/user-attachments/files/16396043/gcode.zip)
After just unload and load. I can print again one print.

**Reply by szimszon** [reply]
Maybe I was able to fix the issue... need further tests.

I raised the nozzle temp from 220 to 223, lowered bed temp from 60 to 55 and speed up the first layer from ~60mm/s to 70mm/s...

**Comment by szimszon** [comment]
No, it did not help :(

![image](https://github.com/user-attachments/assets/57b611ff-2498-446a-9167-07d55d609e2f)

**Comment by bazdmeg949** [comment]
i have a k1 from the first series that now is pretty reliable. 
so i had this issue too, so what i found is the k1 has an insufficient hot end cooler so slow volumetric speeds with pla is a no go (disable ironing for pla), the stock extruder is garbage, the 2nd gen is better but still trash, so upgrade to orbiter v2, and have a top spool holder (eliminate the drag on the filament), and change nozzles sometimes, also a good idea is to clean the filaments way between the nozzle and the extruder (use a drill if needed but carefully)

links to help:
https://www.thingiverse.com/thing:6685761 because otherwise the cable chain would be in the way
https://www.printables.com/model/623172-creality-k1max-orbiter-20-adapter cabling is in the comments
https://www.thingiverse.com/thing:6650979 (i am working on a closed version with runout sensor and automatic drying but its not ready yet)
